deploy/main.tf

```diff
@@ -3,37 +3,58 @@
 //
 
 variable "SITE_URL" {
-  default = "jamaica.infrastructureresilience.org"
+  default = "global.infrastructureresilience.org"
 }
 
 variable "instance_type" {
   description = "AWS EC2 instance type"
-  default = "t3.micro"
+  default     = "t3.micro"
 }
 
 provider "aws" {
   region = "eu-west-2"
 }
 
+// Leave this empty, to be prompted when running terraform apply
+variable "RDS_POSTGRES_PASSWORD" {}
+variable "RDS_MYSQL_PASSWORD" {}
+
 //
 // EC2 Connection
 // Keypair, VPC, Security Group
 //
 
 resource "aws_key_pair" "deployer" {
-  key_name   = "opsis-aws-deployer-jamaica"
-  public_key = "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIAjX5yhc7GWROTVOM8r92rO6MEUyKt/JfTCQzCY/lNi9 opsis-aws-jamaica"
+  key_name   = "opsis-aws-deployer-global"
+  public_key = "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIJi8SKyeEXTv1Mev8BU4iH6xGb3PRyNnDtZbbjQcsHLp opsis-aws-deployer-global"
+}
+
+data "aws_availability_zones" "available" {}
+
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "2.77.0"
+
+  name                 = "global"
+  cidr                 = "10.0.0.0/16"
+  azs                  = data.aws_availability_zones.available.names
+  public_subnets       = ["10.0.4.0/24", "10.0.5.0/24", "10.0.6.0/24"]
+  enable_dns_hostnames = true
+  enable_dns_support   = true
 }
 
-resource "aws_default_vpc" "jamaica" {
+resource "aws_db_subnet_group" "global" {
+  name       = "global"
+  subnet_ids = module.vpc.public_subnets
+
   tags = {
-    Name = "Jamaica VPC"
+    Name = "Global VPC"
   }
 }
 
 resource "aws_security_group" "access_http_ssh" {
```

Comment:
Do we need both this security group and the one defined from line 85?

Reply:
So this was a left-over from the first deployment which failed to re-deploy due to a known bug with TF relating to removal / alteration of security group names:  https://github.com/hashicorp/terraform-provider-aws/issues/265

I had to include both in order to enable the redeployment.

Realistically this whole file should probably not be in public source control - a seperate point but both should probably be tidied up.

Reply:
Okay - for this and the below, let's check in on Monday, likely leave on the pile for now.